ui/pages/articles/_slug/index.vue

```diff
@@ -4,6 +4,14 @@
       <h1 class="mb-10">{{ article.title }}</h1>
       <div v-if="article.content" id="article-content" v-html="marked(article.content)"></div>
     </v-container>
+    <v-container>
+      <disqus
+        shortname="webdevflow"
+        :identifier="'article-' + article.id + '-v2'"
+        :url="'https://webdevflow.com/articles/' + article.slug + '?comments-version=2'"
```

Comment:
@swilgosz I used configuration similar to your blog, but I'm getting error like:

![Screenshot from 2020-06-11 11-33-07](https://user-images.githubusercontent.com/10059264/84369896-d4849b00-abd7-11ea-9db2-df600d3839d4.png)

Do you think you need to configure something in disqus panel to make it working ? Or maybe it would work automatically after moving into the production domain ?

I also wonder why `v2` suffix was used in `blog` project, and whether I need `?comments-version=2` param in url.

Reply:
Yes, I need to allow to load disqus by setting the api key. Thanks, that's enough, I can tackle that from here. About the V2 - i needed to do that, as I had messed up the first implementation I did and had the same commmens thread in all articles.